### PR TITLE
Lpad rpad

### LINKED ISSUE #269: Function: LPAD(), RPAD()
- `LPAD(string TEXT, length INTEGER, fill TEXT) -> TEXT`
  - Add `fill`(a space by default) to the left of `string`. If `string` is longer than `length`, it shorten to `length`.
- `RPAD(string TEXT, length INTEGER, fill TEXT) -> TEXT`
  - Add `fill`(a space by default) to the right of `string`.  If `string` is longer than `length`, it shorten to `length`.

### Examples
1. If `string` is longer than `fill`
``` sql
SELECT LPAD('hello', 10, 'ab'), RPAD('hello', 10, 'ab')
```

|lpad|rpad|
|-|-|
|ababahello|helloababa|

2. If `string` is shorter than `fill`
``` sql
SELECT LPAD('hello', 3, 'ab'), RPAD('hello', 3, 'ab')
```

|lpad|rpad|
|-|-|
|hel|hel|

3. If `fill` is not provided
``` sql
SELECT LPAD('hello', 10), RPAD('hello', 10)
```

|lpad|rpad|
|-|-|
|&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;hello|hello&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|

### PR DESCRIPTION
**Pull request by No-YE**
resolve #269 

Implement Lpad and Rpad functions.
- ` LPAD(string TEXT, length INTEGER, fill TEXT) -> TEXT`
  - Add `fill`(a space by default) to the left of `string`. If `string` is longer than `length`, it shorten to `length`.
- `RPAD(string TEXT, length INTEGER, fill TEXT) -> TEXT`
  - Add `fill`(a space by default) to the right of `string`. If `string` is longer than `length`, it shorten to `length`.

### CODE REVIEW

File: src/executor/evaluate/mod.rs [hunk 1]
```diff
@@ -300,6 +300,59 @@ async fn evaluate_function<'a, T: 'static + Debug>(
...
+                    return Err(EvaluateError::FunctionRequiresIntegerValue(name.to_owned()).into());
+                }
+            };
+
+            let fill = match fill {
+                Some(expr) => match eval_to_str(name, expr).await? {
+                    Nullable::Value(v) => v,
+                    Nullable::Null => {
+                        return Ok(Evaluated::from(Value::Null));
+                    }
```
Comment (panarch) [inline src/executor/evaluate/mod.rs:334]: There are three `Ok(Evaluated::from(Value::Null)` and a single `Err(..)` and these are not tested.
We need to have test cases for them.

**Review by panarch — CHANGES_REQUESTED** [review]
Implementation codes look really, really good, thanks! 👍
There's an only single request about adding more test cases.

**Comment by No-YE** [comment]
@panarch Thank you for your review. I added some tests as you said.

**Review by panarch — APPROVED** [review]
Nevertheless there were lots of code conflicts to resolve due to PR merges, thanks for updating so nicely to fit to the new codes.
Thanks!! merging 👍 👍 👍

**Pull request merged**